spec/acceptance/suites/rpm_docker/nodesets/default.yml

```diff
@@ -10,14 +10,25 @@ HOSTS:
     docker_cmd: '/usr/sbin/sshd -D'
     docker_preserve_image: true
     docker_image_commands:
-      # For SELinux Builds
+      # See https://github.com/CentOS/sig-cloud-instance-images/issues/15
+      - 'rm -f /var/lib/rpm/__db*'
+      - 'rpm --rebuilddb'
+      - 'yum clean all'
+      # This is a hack to force an exit code of 0
+      # Sometimes yum-plugin-ovl will install dependencies, which will fail with rpmdb errors
+      # since yum-plugin-ovl is not installed (chicken and egg)
+      - 'yum install -y yum-plugin-ovl || :'
       - 'yum install -y yum-utils'
 
+      # Openssl 1.0.2 required for gem-wrapper and ruby 2.1.0 openssl
+      # Grab it before disabling base
+      - 'yumdownloader openssl openssl-libs openssl-devel --arch=x86_64'
```

Comment:
We can't do this. It upgrades the `libselinux` libraries to a version that are later than we need for building the packages.

Reply:
I'll see if I can re-work it to keep libselinux downgraded. Unfortunately, I don't think we can get around the dependency on 1.0.2.

Reply:
Using F27 base, these packages are not updating libselinux when building CentOS7.  They have no external deps at time of install.  After install, `yum info libselinux` returns version 2.2.2, `yum info policycoreutils` returns version 2.2.5, and `yum info selinux-policy` returns 3.12.1, all of which match the requires in the simp-environment specfile, and it successfully builds for me.

Reply:
@nick-markowski Validated what you indicated. Unsure why I saw it on my one test node, may have been a PEBKAC error.

Reply:
You should no longer need this lines after the updates from @op-ct